OracleAccessManagement/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,447 @@
+	
+	$ docker images
+	
+    REPOSITORY                                                    TAG                 IMAGE ID            CREATED             SIZE
+    container-registry.oracle.com/middleware/fmw-infrastructure   12.2.1.4            760157811a73        4 weeks ago         2.33GB
+    oracle/fmw-infrastructure                                     12.2.1.4.0          760157811a73        4 weeks ago         2.33GB
+    container-registry.oracle.com/java/serverjre                  8                   da8a8a240247        2 months ago        305MB
+    oracle/serverjre                                              8                   da8a8a240247        2 months ago        305MB
+	
+
+	
+### Downloading the OAM Docker files
+
+  1. Make a work directory to place the OAM Docker files:
+     
+	$ mkdir <work directory>
+	
+  2. Download the OAM Docker files from the OAM [repository](https://github.com/oracle/docker-images) by running the following command:
+
+   
+    $ cd <work directory>
+	$ git clone https://github.com/oracle/docker-images
+
+### Downloading the 12.2.1.4.0 Identity Management shiphome and Patches
+
+  1. Download the [Oracle Identity and Access Management 12cPS4 software](https://www.oracle.com/middleware/technologies/identity-management/downloads.html) to a stage directory. Unzip the downloaded `fmw_12.2.1.4.0_idm_Disk1_1of1.zip` file and copy the `fmw_12.2.1.4.0_idm.jar` to `<work directory>/docker-images/OracleAccessManagement/dockerfiles/12.2.1.4.0/`:
+    
+	$ unzip fmw_12.2.1.4.0_idm_Disk1_1of1.zip
+    $ cp fmw_12.2.1.4.0_idm.jar <work directory>/docker-images/OracleAccessManagement/dockerfiles/12.2.1.4.0/fmw_12.2.1.4.0_idm_generic.jar
+	
+   **Note**: The filename must be changed to `fmw_12.2.1.4.0_idm_generic.jar` when copying to the 12.2.1.4.0 directory.	
+   
+  2. Create the following directories under the `12.2.1.4.0` directory:
+	
+	$ mkdir -p <work directory>/OracleAccessManagement/dockerfiles/12.2.1.4.0/patches
+	$ mkdir -p <work directory>/OracleAccessManagement/dockerfiles/12.2.1.4.0/opatch_patch
+	
+  3. View the latest `manifest.<date>.properties` from this [repository](./imagetool/12.2.1.4.0).	
+  
+     Look at the `[XXXX_PATCH]` sections and download the one off patches referenced. For example, the `manifest.oam.july2020.properties` below shows the following required patches under `[INFRA_PATCH]` and `[OAM_PATCH]`:     
+ 
+    [JDK]
+    jdk-8u261-linux-x64.tar.gz
+
+    [INFRA]
+    fmw_12.2.1.4.0_infrastructure.jar
+
+    [INFRA_PATCH]
+    p28186730_139424_Generic.zip:Opatch
+    p31537019_122140_Generic.zip:WLS
+    p31544353_122140_Linux-x86-64.zip:WLS
+    p31470730_122140_Generic.zip:COH
+    p31488215_122140_Generic.zip:JDEV
+
+
+    [OAM]
+    fmw_12.2.1.4.0_idm_generic.jar
+
+    [OAM_PATCH]
+    p31556630_122140_Generic.zip:OAM
+   
+   
+  6. Download any patches listed in the manifest file from [My Oracle Support](https://support.oracle.com).
```

Comment:
All binaries required to build this image must be available to users without a CSI. Access to MOS should not be required.